Balance Vending Machines

Spircen opened the pull request:
## What Does This PR Do
Agrega costos a los items dentro de HatLord, ShoeLord, ClothesMate, ArtVend y AutoDrobe. Reemplaza el paper por paper bins dentro del ArtVend. Elimina trajes exclusivos del sindicato de las ClothesMate, SuitLord y PrizeCounter. Agrega 1 parche bruto y burn en los NanoMed Emergency, elimina el HA que se encontraba dentro de estos y reduce el Spacellin que venia dentro de ellas de 4 a 2. Baja el total de parches de 10 a 5 dentro de los NanoMed, reduce de 4 a 2 HA y HA upgrade de estas. Reduce de 6 a 3 el total de Syndicakes en maquinas GetMore. Por ultimo baja el costo de la bicicleta de 10000 a 5000 tickets de premios de arcade.

## Why It's Good For The Game
Actualmente la gente disfruta de poder vestir bastante bien sin gastar un billete extra, espero que con estos cambios la gente tenga que comenzar a tener mas en cuenta el dinero dentro del juego para poder pagar por sus servicios, a la par que balanceamos unos cuantos items como dentro de medbay que gozaba anteriormente de una cantidad enorme de parches desde un inicio. Por ultimo elimina vestimenta exclusiva del sindicato la cual nunca debió de estar.

## Images of changes

                                              Precios distintos y Validación de compra
![image](https://user-images.githubusercontent.com/46639834/78301673-abb89900-74f6-11ea-9751-c78bb464abc2.png)

## Changelog
:cl:
tweak: Balance de Expendedoras
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Spircen committed:
Cambios a expendedoras

-Agrega costo a los objetos detro de HatLord, ShoeLord, ClothesMate, ArtVend y AutoDrobe
-Reemplaza paper por paper bin en ArtVend
-Elimina trajes exclusivos del sindicato de ClothesMate, SuitLord y PrizeCounter
-Agrega 1 parche bruto y burn en el NanoMed Emergency
-Elimina los Health Analyzer de los NanoMed Emergency
-Reduce a 2 Spacellin de NanoMed Emergency
-Baja los precios de Xtra
-Baja de 10 a 5 el total de parches en NanoMed
-Baja de 6 a 3 Syndicakes en maquinas
-Baja de 4 a 2 HealthAnalyzer de NanoMed y su mejora
-Baja el costo de bicicleta de 10000 a 5000

N0thy commented:
¿para cuando el PR de economía? paralospobresquenotienendinerito... 😔

Spircen commented:
Para cuando tengamos mayonesa en el juego 

![image](https://user-images.githubusercontent.com/46639834/78307816-0efcf800-7504-11ea-8fed-28b4b7be37b8.png)





No es cierto, no llores ;w;

Tilk1 commented:
Excelente cambio, en cuanto a la cantidad de parches esto debe ser porq en Paradise tienen el triple de gente y es logico que se necesiten mas.

Pulgar arriba para este PR

Evankhell561 commented on code/modules/arcade/prize_datums.dm:
```diff
@@ -204,12 +204,6 @@ var/global/datum/prizes/global_prizes = new
 	typepath = /obj/item/toy/toy_xeno
 	cost = 80
 
-/datum/prize_item/tacticool
```
Por qué quitas este y donde lo encuentras ahora?

Spircen replied:
Lo puedes encontrar como regalo de navidad y dentro de un locker de ropa de gimmick, se supone es ropa especializada del sindicato, como tal únicamente esta aquí por capricho debido a que es el único jumpsuit que no tiene sensores.

Spircen committed:
Merge branch 'master' into Vending-Changes

Spircen committed:
Merge remote-tracking branch 'upstream/master' into Vending-Changes

Spircen committed:
Global-Stuff

Silverexperience commented on code/game/machinery/vending.dm:
```diff
@@ -1703,7 +1820,27 @@
 					/obj/item/clothing/under/color/white = 10, /obj/item/clothing/under/color/yellow = 10,/obj/item/clothing/under/color/lightblue = 10,/obj/item/clothing/under/color/aqua = 10,/obj/item/clothing/under/color/purple = 10,/obj/item/clothing/under/color/lightgreen = 10,
 					/obj/item/clothing/under/color/lightblue = 10,/obj/item/clothing/under/color/lightbrown = 10,/obj/item/clothing/under/color/brown = 10,/obj/item/clothing/under/color/yellowgreen = 10,/obj/item/clothing/under/color/darkblue = 10,/obj/item/clothing/under/color/lightred = 10, /obj/item/clothing/under/color/darkred = 10)
-	contraband = list(/obj/item/clothing/under/syndicate/tacticool = 5,/obj/item/clothing/under/color/orange = 5)
+	contraband = list(/obj/item/clothing/under/color/orange = 5)
```
Para la nueva ropa de contrabando podrias poner ropa rusa, para no simplemente dejar una camiseta naranja que lo unico de ilegal que tiene es que lo usan los prisioneros.

Spircen replied:
Vale, agregue unos trajes de sovieticos

Silverexperience commented on code/game/machinery/vending.dm:
```diff
@@ -1723,6 +1860,15 @@
 	product_ads = "Put your foot down!;One size fits all!;IM WALKING ON SUNSHINE!;No hobbits allowed.;NO PLEASE WILLY, DONT HURT ME- *BZZT*"
 	products = list(/obj/item/clothing/shoes/black = 10,/obj/item/clothing/shoes/brown = 10,/obj/item/clothing/shoes/blue = 10,/obj/item/clothing/shoes/green = 10,/obj/item/clothing/shoes/yellow = 10,/obj/item/clothing/shoes/purple = 10,/obj/item/clothing/shoes/red = 10,/obj/item/clothing/shoes/white = 10,/obj/item/clothing/shoes/sandal=10)
 	contraband = list(/obj/item/clothing/shoes/orange = 5)
```
Tambien podrias añadir aqui mas contrabando.
Y tambien me pareceria buena idea quitar el wand of nothing del autrodrobe que no pega mucho en un dispensador de ropas.

Spircen replied:
Añadere unas jackboots y borrare la wand

Silverexperience replied:
Los jackboots ya salen sin ser contrabando en el autodrobe, si los pones de contrabando quita los del autodrobe.

Spircen committed:
Adds and Comment

Agrega jackboots y soviet a contrabando y elimina wand of nothing

Silverexperience commented:
Tambien si vas a añadir precios a los autodrobe y demas, seria mejor que tenga más que solo 1 objeto, almenos 3 de cada clase.

Spircen committed:
Cambios

-Aumenta de 1 a 3 las cosas del autodrobe y de 1 a 2 algunas de ClothesMate a la par que saca las jackboots de el autodrobe

Spircen commented:
Ahí quedaron los aumentos de 1 a 3 y algunos mas ademas de hacer contraband las jackboots

Silverexperience reviewed (commented):
Ya que estas balanceando los vendigs, seria bueno que quitaras del Xtra las dos cartas de juego,  las cartas del tarot, el patito de goma, los chuchillos y la taza, ya que estos no son accesorios y no pegan con el objetivo del Xtra.

Spircen committed:
Merge branch 'master' into Vending-Changes

Spircen committed:
Sacamos juegos de mesa, patito y taza.

Dejamos los cuchillos por que por el momento no se pueden conseguir de otra forma y esto los sacaria del juego.

A la par ordena el tab extra de algunos objetos

Spircen commented on code/game/machinery/camera/camera.dm:
```diff
@@ -34,8 +34,8 @@
 
-/obj/machinery/camera/Initialize()
-	. = ..()
+/obj/machinery/camera/New()
```
No entiendo por que envía este cambio si en master ya esta de esta forma esta linea de code.

Spircen committed:
Revert "I fucked up"

This reverts commit 6860742d1c037851a400d706320f942e803371b6.

Silverexperience commented:
¿Pero esto no reverteria el fixeo de las camaras?

Spircen commented:
Ya, pero no se por que a mi no me muestra ese archivo. Esto es lo que yo veo: 

![image](https://user-images.githubusercontent.com/46639834/79397172-19c37e00-7f43-11ea-834a-853c3568fae0.png)

¿Hago el revert al revert?

Silverexperience commented:
> 
> 
> Ya, pero no se por que a mi no me muestra ese archivo. Esto es lo que yo veo:
> 
> ![image](https://user-images.githubusercontent.com/46639834/79397172-19c37e00-7f43-11ea-834a-853c3568fae0.png)
> 
Intenta hacer el revert al revert, quizas no vuelva a salir ese archivo.

Spircen committed:
Revert "Revert "I fucked up""

This reverts commit 3512ddf86757df042e2e73c6d0d0daa197e2db2b.

Spircen commented:
Ahí quedo, pero no entiendo por que pasa la verdad. Quiero decir ese archivo ya esta igual que el live.

Spircen committed:
Merge branch 'master' into Vending-Changes

Spircen committed:
Avoid Conflict

Spircen committed:
Merge remote-tracking branch 'upstream/master' into Vending-Changes

Spircen committed:
Merge branch 'master' into Vending-Changes
